go.sum

```diff
@@ -1324,6 +1326,12 @@ github.com/gosimple/unidecode v1.0.1/go.mod h1:CP0Cr1Y1kogOtx0bJblKzsVWrqYaqfNOn
 github.com/gosnmp/gosnmp v1.34.0 h1:p96iiNTTdL4ZYspPC3leSKXiHfE1NiIYffMu9100p5E=
 github.com/gosnmp/gosnmp v1.34.0/go.mod h1:QWTRprXN9haHFof3P96XTDYc46boCGAh5IXp0DniEx4=
 github.com/gotestyourself/gotestyourself v2.2.0+incompatible/go.mod h1:zZKM6oeNM8k+FRljX1mnzVYeS8wiGgQyvST1/GafPbY=
+github.com/grafana/blackbox_exporter v0.21.1-0.20220804123005-e102fa2a0e07 h1:79RulSWptJKISfYW0WvPqJkQ1rA5Ar+mziK9qWvxdcs=
+github.com/grafana/blackbox_exporter v0.21.1-0.20220804123005-e102fa2a0e07/go.mod h1:5nS7A7JCcK9rLGMmvkTU76NxeIn1Ga42AByUJnFCphU=
+github.com/grafana/blackbox_exporter v0.21.1-0.20220808114229-1b1ccc2482d8 h1:IHaC/acatFJsVnPsA6tp9HcQvvRYrMICzXFPyhg6g/o=
+github.com/grafana/blackbox_exporter v0.21.1-0.20220808114229-1b1ccc2482d8/go.mod h1:5nS7A7JCcK9rLGMmvkTU76NxeIn1Ga42AByUJnFCphU=
+github.com/grafana/blackbox_exporter v0.21.1-0.20220808121409-717fd374cd41 h1:LJt/6gBIeV1KN8v6zH25N7jE7aC6aylZOPYp7v4LjGc=
+github.com/grafana/blackbox_exporter v0.21.1-0.20220808121409-717fd374cd41/go.mod h1:5nS7A7JCcK9rLGMmvkTU76NxeIn1Ga42AByUJnFCphU=
```

Comment:
Let's run `go mod tidy` to remove the multiple references and only keep the one actually used.

I _think_ the change here is commit https://github.com/grafana/blackbox_exporter/commit/717fd374cd41 where you exported some functionality, right? Is this change available in any branch, so that we can build on it if needed?

Reply:
I'm waiting for this [PR](https://github.com/prometheus/blackbox_exporter/pull/957) to be merged. I can run `go mod tidy` after.